packages/auth/rollup.config.ts

```diff
@@ -9,15 +9,14 @@ const { modules, umd } = require('../../rollup-config-generator');
 const pkg = require('./package.json');
 
 export default [
-  umd('auth', 'src/index.ts', [
+  umd('CartoToolkitAuth', 'src/index.ts', pkg, [
```

Comment:
Why do we need to add the pkg here?

Reply:
I've changed the config generator, it uses the package to extract the name of the module / browser / main files.